FindMe/Parse Manager/DatabaseManager.m

```diff
@@ -33,23 +35,24 @@ + (void)saveUser:(User *)user withCompletion:(void(^)(NSError *error))completion
     }];
 }
 
-+ (User *)getCurrentUser
++ (void)currentUser:(void(^)(User *user))completion
 {
     PFUser *user = [PFUser currentUser];
     User *currentUser = [User new];
     currentUser.firstName = user[@"firstName"];
     currentUser.lastName = user[@"lastName"];
     currentUser.email = user[@"email"];
     currentUser.username = user[@"username"];
-    PFFileObject *userProfileImage = user[@"profileImage"];
+    PFFileObject *userProfileImage = user[@"profilePhoto"];
```

Comment:
Can add string constants at the top of this file to avoid possible mistakes from changing string keys.